go.mod

```diff
@@ -157,6 +157,7 @@ require (
 	github.com/storageos/go-api v0.0.0-20180912212459-343b3eff91fc
 	github.com/stretchr/testify v1.2.2
 	github.com/syndtr/gocapability v0.0.0-20160928074757-e7cb7fa329f4 // indirect
+	github.com/tools/godep v0.0.0-20180126220526-ce0bfadeb516 // indirect
```

Comment:
drop this commit

Reply:
Yeah, this shouldn't be included in the PR. Is the change needed at all?

Reply:
@misterikkit @liggitt Squashed and rebased. Dropped commits as per recommendation